core/src/main/scala/cats/MonoidK.scala

```diff
@@ -38,3 +47,13 @@ import simulacrum.typeclass
       def combine(x: F[A], y: F[A]): F[A] = self.combine(x, y)
     }
 }
+
+trait CompositeMonoidK[F[_],G[_]]
+  extends MonoidK[λ[α => F[G[α]]]] {
+
+  implicit def F: MonoidK[F]
+  implicit def G: MonoidK[G]
```

Comment:
I'm not sure I understand this comment. I'm saying that I think we should be able to change `MonoidK[F] compose MonoidK[G]` to simply `MonoidK[F].compose[G]` since the `MonoidK[G]` is ignored as far as I can tell. I might be missing something.